Cargo.toml

```diff
@@ -121,33 +121,33 @@ iced_widget = { version = "0.12", path = "widget" }
 iced_winit = { version = "0.12", path = "winit" }
 
 async-std = "1.0"
-bitflags = "1.0"
+bitflags = "2.4"
```

Comment:
No need to specify the minor version for stable crates unless we depend on added features: `2.0` should just work.